XpringKit/PayID/PayIDClient.swift

```diff
@@ -3,32 +3,42 @@ import Foundation
 
 /// Implements interaction with a PayID service.
 /// - Warning:  This class is experimental and should not be used in production applications.
-public class PayIDClient: PayIDClientProtocol {
+public class PayIDClient {
   /// The network this PayID client resolves on.
-  public let network: XRPLNetwork
+  private let network: String
 
-  /// Initialize a new PayIDClient.
+  /// Initialize a new PayID client.
   ///
   /// - Parameter network: The network that addresses will be resolved on.
-  public init(network: XRPLNetwork) {
+  ///
+  /// - Note: Networks in this constructor take the form of an asset and an optional network (<asset>-<network>).
+  /// For instance:
+  ///   - xrpl-testnet
+  ///   - xrpl-mainnet
+  ///   - eth-rinkby
```

Comment:
nit: `eth-rinkeby` ?

Reply:
nice catch Dino - I've made a note for myself to make sure this is corrected anywhere else this comment appears in the SDKs, since I originally stole this comment from Keefer.